rb/lib/selenium/webdriver/common/selenium_manager.rb

```diff
@@ -77,14 +74,12 @@ def binary
         end
 
         def validate_location(location)
+          raise Error::WebDriverError('Unable to locate or obtain Selenium Manager binary') if value.nil?
+
           begin
-            Platform.assert_file(location)
             Platform.assert_executable(location)
-          rescue TypeError
-            raise Error::WebDriverError,
-                  "Unable to locate or obtain Selenium Manager binary; #{location} is not a valid file object"
           rescue Error::WebDriverError => e
-            raise Error::WebDriverError, "Selenium Manager binary located, but #{e.message}"
+            raise Error::WebDriverError, "#{exe} located, but has problems: #{e.message}"
```

Comment:
Do we have `exe` variable in this context?

Reply:
Nope, bad copy paste.